openstack/designate/templates/seeds.yaml

```diff
@@ -938,3 +938,42 @@ spec:
       role: cloud_dns_admin
       inherited: true
 {{- end }}
+
+{{- if .Values.nanny_enabled }}
+---
+apiVersion: "openstack.stable.sap.cc/v1"
+kind: "OpenstackSeed"
+metadata:
+  name: designate-nanny-seed
```

Comment:
would you like to replace designate here with {{ .Release.Name }} ?
In that case seeds for global case will be called different, automatically.

Reply:
oh, makes sense. the name then will be designate-global-nanny which is not in sync with the general naming scheme of designate-nanny-global.  But will change it, makes sense!